test/boulder-config.json

```diff
@@ -172,5 +172,5 @@
     "dnsTimeout": "10s"
   },
 
-  "subscriberAgreementURL": "http://localhost:4300/terms"
+  "subscriberAgreementURL": "http://localhost:4000/directory"
```

Comment:
Surely this isn't right. But what _is_?

At least this passes the test given in README without ECONNREFUSED or an infinite redirect crash...